src/DependencyInjection/CsaGuzzleExtension.php

```diff
@@ -95,18 +96,12 @@ private function processCacheConfiguration(array $config, ContainerBuilder $cont
     private function processClientsConfiguration(array $config, ContainerBuilder $container, Definition $clientFactory, Definition $descriptionFactory)
     {
         foreach ($config['clients'] as $name => $options) {
-            $clientFactory->addMethodCall('registerClientConfiguration', [
-                $name,
-                $options['config'],
-                $options['subscribers']
+            $client = new Definition($config['factory_class']);
+            $client->addArgument($options['config']);
+            $client->addTag(SubscriberPass::CLIENT_TAG, [
+                'subscribers' => implode(', ', $options['subscribers'])
```

Comment:
I've been quite busy for the past week unfortuntely :/

Also, there's a slight problem with being able to turn subscribers off (the `'cache' => false` case). There's no nice way of translating that directly to the current tag syntax. I was thinking of explicitly listing all subscribers in that case, but never had the time to implement it.